cmd/cstor-pool-mgmt/controller/replica-controller/handler.go

```diff
@@ -132,15 +132,21 @@ func (c *CStorVolumeReplicaController) cVREventHandler(operation common.QueueOpe
 			return string(apis.CVRStatusDeletionFailed), err
 		}
 		return "", nil
+	case common.QOpModify:
+		fallthrough
 	case common.QOpSync:
-		glog.Infof("Synchronizing CstorVolumeReplica status for volume %s", cVR.ObjectMeta.Name)
+		if operation == common.QOpModify {
```

Comment:
I fail to understand this? How will operation be Modify inside Sync condition?

Reply:
Basically, for logging purpose, we are treating both sync and modify operation as same. The above `case common.QOpModify` will `fallthrough` to the sync condition. i.e as of now we are not really handeling the cvr modify event.